ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/ShowPopup/README.md

```diff
@@ -0,0 +1,37 @@
+# Show a popup
+
+Show predefined popups from a web map.
+
+![](screenshot.png)
+
+## Use case
+
+Many web maps contain predefined popups which are used to display the attributes associated with each feature layer in the map, such as hiking trails, land values, or unemployment rates. You can display text, attachments, images, charts, and web links. Rather than creating new popups to display information, you can easily access and display the predefined popups.
+
+## How to use the sample
+
+Tap on the features to prompt a popup that displays information about the feature.
+
+## How it works
+
+1. Create and load an `Map` using a URL.
+2. Set the map to a `MapView` and connect to the `MapView.onMouseClicked` signal.
```

Comment:
nit picky, but technically, "mouseClicked" is the signal name. "onMouseClicked" is the signal handler name